Article/Article.js

```diff
@@ -3,24 +3,36 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = document.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.innerText = "expand";
+
     // Set a click handler on the expandButton reference, calling the expandArticle method.
+
+
     
   }
 
   expandArticle() {
+      // article.classList.toggle(".menu--open");
+      article.addEventListener('click',expandButton);
+    }
+
+    
```

Comment:
You're well on your way with this! Remember to push your commits up so it's completely updated